000/0001/tst.py

```diff
@@ -0,0 +1,36 @@
+# If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these \
+# multiples is 23.
+# Find the sum of all the multiples of 3 or 5 below 1000.
+
+
+def main():
+
+    my_list3 = []
+    my_list5 = []
+
+    number1 = 3
+    number2 = 5
+
+    for a in range(1000):
+        result = a * number1

```

Comment:
This is not what description asked for. 
It asked for numbers that can be divided by 3 or 5.